tests/network-tests/start-storage.sh

```
@@ -1,18 +1,22 @@
 set -e
 
+source .env
+
 TMP=$0
-THIS_DIR=`dirname $TMP`
+THIS_DIR=$(dirname $TMP)
```

The back tick is better, so the script exits with error if the invoked command errors. 
With the `$()` any errors are hidden and the script continues, with the variable assiged being empty, which most times is not the behavior we want.